core/storage/question/gae_models.py

```diff
@@ -54,6 +54,9 @@ class QuestionModel(base_models.VersionedModel):
         required=True, indexed=True)
     # The ISO 639-1 code for the language this question is written in.
     language_code = ndb.StringProperty(required=True, indexed=True)
+    # The skill ids linked to this question.
+    linked_skill_ids = ndb.JsonProperty(
```

Comment:
Should be ndb.StringProperty right?

Why required=False?

Should also have indexed=True

Reply:
I had kept required=False so that it doesn't break production for questions that are already in the backend